rcgen/tests/webpki.rs

```diff
@@ -590,17 +593,9 @@ fn test_webpki_crl_parse() {
 	let der = crl.serialize_der_with_signer(&issuer, &issuer_key).unwrap();
 
 	// We should be able to parse the CRL DER without error.
-	let webpki_crl = BorrowedCertRevocationList::from_der(&der).expect("failed to parse CRL DER");
-
-	// Webpki represents certificate SPKIs internally without the outer SEQUENCE.
-	// We remove that here before calling verify_signature.
-	let issuer_spki = issuer_key.public_key_der();
-	let raw_spki = yasna::parse_der(&issuer_spki, |reader| reader.read_tagged_der()).unwrap();
-
-	// We should be able to verify the CRL signature with the issuer's raw SPKI.
-	webpki_crl
-		.verify_signature(&[&webpki::ECDSA_P256_SHA256], raw_spki.value())
-		.expect("failed to validate CRL signature");
```

Comment:
just noting for other reviewers that this hunk of test code falls away because we removed the `verify_signature` fn of the CRL type upstream. It didn't mesh well with the existing API surface, was difficult to use (as evidenced by the SPKI SEQUENCE dance), and had no other consumers we were aware of except this unit test.

The `test_webpki_crl_revoke` test that remains uses the CRL for a revocation checking decision and the webpki crate will verify the signature as part of that work.